exampleCourse/questions/element/hiddenHints/question.html

```diff
@@ -22,15 +25,24 @@
       Another hint that will be shown immediately.
     </pl-hint>
 
+    <!-- Comments are allowed inside the hints element -->
+
     <pl-hint show-after-submission=5 hint-name="Final Hint">
       This is a hint that will be shown after <b>five</b> incorrect submissions for the current question.
     </pl-hint>
 
-    <!-- Comments are allowed inside the hints element -->
-
     <pl-hint show-after-submission=3>
       This is a hint that will be shown after <b>three</b> incorrect submissions for the current question.
     </pl-hint>
+
+
+    <!-- Specific Wrong Answer Hints, this value is computed on server.py -->
```

Comment:
As a general rule, in recent changes we have been encouraging instructor comments in question.html to [use Mustache comments](https://prairielearn.readthedocs.io/en/latest/question/template/#hiding-staff-comments-in-questionhtml). This shouldn't matter much in this context, but might as well do that here anyway.

Also, it's probably better to be more verbose to clarify that both changes are needed to achieve this result. Suggestion:

```suggestion
    {{! To show hints conditionally based on previous answers, you may use mustache conditionals. Note that this requires a related computation in server.py as well. }}
```

Reply:
Bonus "points" if you also update the other comments in this file.